integration-tests/conftest.py

```diff
@@ -1,10 +1,27 @@
 import pytest
+import os
+import subprocess
 
 
+@pytest.fixture(scope="session", autouse=True)
+def install_katello_rpm(test_config):
+    if "satellite" in os.getenv('ENV_FOR_DYNACONF'):
+        # install katello rpm before register system against Satellite
+        satellite_hostname = test_config.get("candlepin", "host")
+        cmd = 'rpm -qa | grep katello | xargs rpm -e;rpm -Uvh http://%s/pub/katello-ca-consumer-latest.noarch.rpm' %satellite_hostname
```

Comment:
Please split this command in two invocations:
- one that removes the old packages
- another one that installs the new one

This way it is easier to check that each command succeeded.